multiversx/mx-api-service
# account token collection roles as inner structure

## Proposed Changes
-  add roles property to accounts/:account/roles/collections/:collection
-  add roles property to accounts/:account/roles/tokens/:token

## How to test
API
-  `accounts/erd1qqqqqqqqqqqqqpgq6xtqy9lda5cj5ts8kfvd8r89wy7n2dn58sts6dguut/roles/collections/MMAF-63c85a` -> should contains roles attribute:
`roles": {
        "canCreate": true,
        "canBurn": true,
        "canAddUri": false,
        "canUpdateAttributes": false
 }`

- `accounts/erd1qqqqqqqqqqqqqpgqnqvjnn4haygsw2hls2k9zjjadnjf9w7g2jpsmc60a4/roles/tokens/MEX-455c57` -> should contains roles attributes:
`roles: {
          "canLocalMint": true,
          "canLocalBurn": true
}`

- `accounts/erd1k4u77ve67r0ve8ywyjaq05pp8y59x683gudfffrwd64q56843kgsyld36e/collections/XMAXIM1-63599d` -> should contain roles attribute

GraphQL
- create query using `tokenWithRolesForAddress` and verify if roles property is defined
